goagen/gen_app/test_generator.go

```diff
@@ -59,9 +60,8 @@ func (g *Generator) generateResourceTest() error {
 		return nil
 	}
 	funcs := template.FuncMap{
-		"isSlice":       isSlice,
-		"initArguments": initArguments,
-		"tempvar":       tempvar,
+		"isSlice": isSlice,
+		"escape":  escape,
```

Comment:
The way that `TestMethod` has `escape` as a method might be better. In this way, we won't have to pass `$test` everywhere in the template.